doc/developer-guide/api/functions/TSSslContext.en.rst

```diff
@@ -43,6 +45,9 @@ server :arg:`name`.
 created from :file:`ssl_multicert.config` matchin against the server
 :arg:`address`.
 
+:func:`TSSslOCSPCallbackSet` sets the OCSP callback described in ATS
```

Comment:
"described in ATS"? I do not understand what that means.

Also, you should use `|TS|` when possible instead of the literal string "ATS".